lib/internal/modules/esm/module_job.js

```diff
@@ -143,7 +143,14 @@ class ModuleJob {
     await this.instantiate();
     const timeout = -1;
     const breakOnSigint = false;
-    await this.module.evaluate(timeout, breakOnSigint);
+    try {
+      await this.module.evaluate(timeout, breakOnSigint);
+    } catch (e) {
+      decorateErrorStack(e);
+
+      // console.log(e)
+      // console.log("FOOOOOOOOOOOOOOOOOOOOOOOOOOOO")
```

Comment:
This should help with finding the various places you need to touch.
```suggestion
      console.log('✉️');
      console.log(e.message);
      console.log('📚');
      console.log(e.stack);
      console.log('🛑');
      console.log(e);
```

Locally that gets me the following output

```
$ ./node foo.mjs
✉️
require is not defined
📚
ReferenceError: require is not defined
    at file:///Users/mylesborins/code/node/main/foo.mjs:7:14
    at ModuleJob.run (node:internal/modules/esm/module_job:153:25)
    at async Loader.import (node:internal/modules/esm/loader:166:24)
    at async Object.loadESM (node:internal/process/esm_loader:68:5)
🛑
ReferenceError: require is not defined
    at file:///Users/mylesborins/code/node/main/foo.mjs:7:14
    at ModuleJob.run (node:internal/modules/esm/module_job:153:25)
    at async Loader.import (node:internal/modules/esm/loader:166:24)
    at async Object.loadESM (node:internal/process/esm_loader:68:5)
```

Reply:
Ah, this makes sense, and what I am getting. For whatever reason, I expected it to print differently. I have what I need to make this error better and maybe write a test.